app/Repositories/SignupRepository.php

```diff
@@ -24,6 +24,7 @@ public function create($data)
         $signup->quantity_pending = isset($data['quantity_pending']) ? $data['quantity_pending'] : null;
         $signup->why_participated = isset($data['why_participated']) ? $data['why_participated'] : null;
         $signup->source = isset($data['source']) ? $data['source'] : null;
+        $signup->competition = isset($data['competition']) ? $data['competition'] : null;
```

Comment:
Totally subjective, but it might make sense to use mass-assignment for these fields:

```php
$signup = new Signup($data);

if (isset($data['created_at'])) {
    // ...
```